src/test/java/org/folio/gobi/HelperUtilsTest.java

```diff
@@ -70,93 +69,47 @@ public void testExtractVendorUidWithMultipleVendors() {
 
   @Test
   public final void testReadJsonAndCreateDefaultMapping() throws Exception {
-	  
-	//Assuming we will get a json object below is test to extract default mappings from json file
-	  /*
-		{
-		    "orderMappings":[
-	        {
-	            "orderType":"ListedPrintMonograph",
-	            "mappings":[{
-	                "field":"ACCOUNT_NUMBER",
-	                "dataSource":{
-	                    "from":"//SubAccount",
-	                    "default":"0"
-	                }
-	            }]
-	        }]
-		}
-	   */
-	  Map<String, Object> dataSourceMap1 = new LinkedHashMap<String, Object>();
-	  dataSourceMap1.put("from", "//SubAccount");
-	  dataSourceMap1.put("default", "0");
-	  JsonObject dataSourceObj1 = new JsonObject(dataSourceMap1);
-	  JsonObject mappingsObj1 = new JsonObject();
-	  mappingsObj1.put("field", "ACCOUNT_NUMBER");
-	  mappingsObj1.put("dataSource", dataSourceObj1);
-	  JsonArray mappingsArray1 = new JsonArray();
-	  mappingsArray1.add(mappingsObj1);
-	  JsonObject orderMappingsObj1 = new JsonObject();
-	  orderMappingsObj1.put("orderType", "ListedPrintMonograph");
-	  
-	  /*
-		{
-	    	"orderMappings":[
-	        {
-	            "orderType":"ListedPrintMonograph",
-	            "mappings":[{
-					"field": "ACQUISITION_METHOD",
-					"dataSource": {
-						"default": "Purchase at Vendor System"
-					}
-	        	}]
-			}]
-		}
-	   */
-	  Map<String, Object> dataSourceMap2 = new LinkedHashMap<String, Object>();
-	  dataSourceMap2.put("default", "Purchase at Vendor System");
-	  JsonObject dataSourceObj2 = new JsonObject(dataSourceMap2);
-	  JsonObject mappingsObj2 = new JsonObject();
-	  mappingsObj2.put("field", "ACQUISITION_METHOD");
-	  mappingsObj2.put("dataSource", dataSourceObj2);
-	  JsonArray mappingsArray2 = new JsonArray();
-	  mappingsArray2.add(mappingsObj2);
-	  
-	  orderMappingsObj1.put("mappings", mappingsArray1);
-	  
-	  JsonArray orderMappingsArray1 = new JsonArray();
-	  orderMappingsArray1.add(orderMappingsObj1);
-	  JsonObject Obj = new JsonObject();
-	  
-	  Obj.put("orderMappings", orderMappingsArray1);
-  }
-  
-  @Test
-  public final void testExtractOrderMappingWithOtherField() throws Exception {
```

Comment:
Isn't this test method required?

Reply:
This functionality has been moved out of HelperUtils and merged with duplicate/similar functionality in MappingHelper, so I removed these test cases.  The code in MappingHelper is already covered by existing tests.